lib/models/message.dart

```diff
@@ -0,0 +1,27 @@
+import './model.dart';
+
+class Message implements Model {
+  final int id;
+  final String message;
```

Comment:
Does message need a src and dest addr? @s-axford

Reply:
Group id at least

Reply:
yeah im not sure exactly what it would need. I'm just thinking from a routing persepective.

Reply:
Might be better to have connection stuff in a different model maybe device which has id, device id, and other bluetooth info

Reply:
see my comment on user.dart review